### PR TITLE
Checking that there are znodes in network

### PR DESCRIPTION
**Pull request by nopslide**
Otherwise, it will prevent setgernerate command to create blocks

### CODE REVIEW

**Comment by reubenyap** [comment]
Not needed according to @psolstice

**Comment by nopslide** [comment]
I think we need to take care in case Zcoin doesn't have any znodes running, and it will preventing solo miner to run command for creating blocks here.

File: src/miner.cpp [hunk 1]
```diff
@@ -1129,10 +1129,15 @@ void static ZcoinMiner(const CChainParams &chainparams) {
...
-                                chainActive.Height() < params.nZnodePaymentsStartBlock ||
-                                mnodeman.GetNextZnodeInQueueForPayment(chainActive.Height(), true, nCount);
+                        // Check you actually have znodes in network
+			            if(mnodeman.CountEnabled() > 0){
+                        	fHasZnodesWinnerForNextBlock =
+                                	params.IsRegtest() ||
+                                	chainActive.Height() < params.nZnodePaymentsStartBlock ||
+                                	mnodeman.GetNextZnodeInQueueForPayment(chainActive.Height(), true, nCount);
+                        }else{
+				            fHasZnodesWinnerForNextBlock = true; // we don't have Znode winner causes we don't have Znode on network
```
Comment (thebevrishot) [inline src/miner.cpp]: Please use new flag instead. Setting `fHasZnodesWinnerForNextBlock` as true isn't make sense.

Comment (riordant) [reply src/miner.cpp]: @thebevrishot added extra param, please re-review

**Comment by qashban4** [comment]
As per project meeting discussion on 23-09-2019: This issue is irrelevant as we are upgrading to core 0.14, so this issue will be closed.

**Pull request closed without merging**